tests/test_threadpoolctl.py

```diff
@@ -302,6 +302,8 @@ def test_nested_prange_blas(nthreads_outer):
     assert original_info == _threadpool_info()
 
 
+# the method `get_original_num_threads` raises a UserWarning in one CI job. It
+# is expected so we can safely filter it.
```

Comment:
Could you please be more precise? Which warning? What CI configuration is causing it?